Source/WebCore/platform/audio/cocoa/CAAudioStreamDescription.h

```diff
@@ -99,3 +100,7 @@ inline CAAudioStreamDescription toCAAudioStreamDescription(const AudioStreamDesc
 }
 
 }
+
+namespace IPC {
```

Comment:
To layer the code correctly, the direct references to IPC namespace must happen only from WebKit level.

This means that this should go int WebCoreArgumentCodersCocoa.h or WebCoreArgumentCoders.h in cocoa-specific ifdef

I know currently there are the IPC friend declarations, those should eventually be removed..
Also likely ArgumentCoder.h was moved into wrong place, WTF. Hopefully these can be moved back.